### ISSUE TITLE
whichenv executable

### ISSUE THREAD

**Issue by rsgalloway** [rsgalloway/envstack#9]
create a `whichenv` executable that wraps `envstack -t`:

```bash
$ whichenv HELLO
HELLO: /path/to/source/stack.env
```

**Comment by rsgalloway** [comment]
this is basically just an alias for `envstack -r HELLO`

```bash
alias whichenv='envstack -t'
$ whichenv HELLO
HELLO: /mnt/homes/rsg/dev/envstack/stack.env
```

**Comment by rsgalloway** [comment]
available in version 0.7.1
